### DISCUSSION TITLE
Features: New Store Apps

### DISCUSSION THREAD

**Discussion by Protektor-Desura** [IceWhaleTech/CasaOS#238] [Ideas]
I would recommend dropping Jackett in favor of Prowlarr instead. It will automatically setup all your indexers in Radarr, Sonarr, Lidarr, Readarr and Mylar3. Prowlarr also supports Usenet and Bittorrent indexers in a single program.
https://github.com/Prowlarr/Prowlarr
https://hub.docker.com/r/linuxserver/prowlarr

Another app you might want to consider is ARM (Automatic Ripping Machine). It allows users who don't know anything to automatically rip their media discs to the server. Just drop the disc in and wait for it to eject and drop the next one in.
https://github.com/automatic-ripping-machine/automatic-ripping-machine
https://hub.docker.com/r/1337server/automatic-ripping-machine

If you want to automatically download music videos then I suggest this script.
https://github.com/RandomNinjaAtk/docker-amvd
https://hub.docker.com/r/randomninjaatk/amvd

If you want to automatically download movie trailers and extras then I suggest this script.
https://github.com/RandomNinjaAtk/docker-amtd
https://hub.docker.com/r/randomninjaatk/amtd

If you wanted to automatically download music from Deezer then you might want to look at this script. It is a little more difficult though because you have to pull your Deezer login cookie/ARL from your browser.
https://github.com/RandomNinjaAtk/docker-amd
https://hub.docker.com/r/randomninjaatk/amd

If you are pushing Open-Source options then I would mention that it would disqualify Plex and Emby and leave only Jellyfin for a media server. Plex and Emby are both commercial and closed source applications.
https://jellyfin.org/
https://github.com/jellyfin/jellyfin
https://hub.docker.com/r/linuxserver/jellyfin

If you are going to push Jellyfin as the media server, then you can drop Ombi in favor of Jellyseerr, which does the same type of thing, letting users request things.
https://github.com/Fallenbagel/jellyseerr
https://hub.docker.com/r/fallenbagel/jellyseerr

If you are going to push Jellyfin then you might want to include Embystat stats program for Jellyfin. 
https://github.com/mregni/EmbyStat
https://hub.docker.com/r/linuxserver/embystat

I would recommend Lidarr being added to the store for music support.
https://lidarr.audio/
https://github.com/Lidarr/Lidarr
https://hub.docker.com/r/linuxserver/lidarr

I would recommend Bazaarr for automatic downloading of missing subtitles.
https://www.bazarr.media/
https://github.com/morpheus65535/bazarr
https://hub.docker.com/r/linuxserver/bazarr

I would recommend Mylar3 for comic books.
https://github.com/mylar3/mylar3
https://hub.docker.com/r/linuxserver/mylar3

I would recommend Readarr for ebooks or audiobooks.
https://readarr.com/
https://github.com/Readarr/Readarr
https://hub.docker.com/r/linuxserver/readarr

**Comment by AlexFigas** [comment]
👑

**Comment by Lauren-ED209** [comment]
I've added this request to the development schedule. Thanks!
